modules/javafx.graphics/src/main/native-prism-sw/JAbstractSurface.c

```diff
@@ -184,29 +189,3 @@ initializeSurfaceFieldIds(JNIEnv* env, jobject objectHandle) {
 
     return retVal;
 }
-
-static void
-disposeNativeImpl(JNIEnv* env, jobject objectHandle) {
```

Comment:
Since you removed `disposeNativeImpl`, you can also remove the static declaration on [line 40](https://github.com/openjdk/jfx/blob/0591e41f9106e93ae957cb9d80cf2e9e2c6fd805/modules/javafx.graphics/src/main/native-prism-sw/JAbstractSurface.c#L40).

Reply:
I shall do this in next commit.